index.html

```diff
@@ -21,7 +21,26 @@
         <title>
             %APP_TITLE%
         </title>
-        <link href="https://api.fontshare.com/v2/css?f[]=nunito@300,700,400&f[]=bebas-neue@400&display=swap" rel="stylesheet">
+        <link href="https://fonts.googleapis.com/css2?family=Assistant:wght@300;400;600;700&family=Manrope:wght@300;400;600;700&display=swap" rel="stylesheet">
+        <style>
+            html, body {
+                margin: 0;
+                padding: 0;
+            }
+
+            body {
+                font-family: "Manrope", sans-serif;
+                font-optical-sizing: auto;
```

Comment:
what does this do?

Reply:
i have taken reference from the Timur and base-react-app, so i don't know exactly why this css are are we need to discuss about it with ankit dai.